op-batcher/flags/flags.go

```diff
@@ -99,6 +99,31 @@ var (
 			return nil
 		},
 	}
+	CompressionAlgoFlag = &cli.StringFlag{
```

Comment:
Please follow the pattern used by the "DataAvailabilityTypeFlag" to make this an enum type flag instead of a string flag.